### PR TITLE
[12.0][MIG] purchase_request_department

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

Depends on:
- [x]  https://github.com/OCA/purchase-workflow/pull/720

CC @Eficent

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional tests good

**Comment by ps-tubtim** [comment]
LGTM.

**Review by ps-tubtim — APPROVED** [review]

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 account-invoicing
 server-tools
 server-ux
+purchase-workflow https://github.com/Eficent/purchase-workflow.git 12.0-mig-purchase_request
```
Comment (LoisRForgeFlow) [inline oca_dependencies.txt]: dependency not merged

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]

File: oca_dependencies.txt (hunk 1 above)
Comment (HviorForgeFlow) [reply oca_dependencies.txt]: @lreficent dependency merged

**Review by LoisRForgeFlow — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**